### PR TITLE
Scheduled Profiler - Add event subscriber to link help page with schedules + profiling status on the schedules page

### PR DESCRIPTION
**Pull request by BazookaMusic**
#### Summary
This PR adds two features:
1) It adds an event subscriber to link the new help page link with the profiler schedules page
2) It adds a new group to show the scheduled profiling status on the schedules page. This new group contains information about if profiling is active for the current session and shows the active schedule.

#### Work Item(s)
Fixes [AB#501774](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/501774)

### CODE REVIEW

**Comment by BazookaMusic** [comment]
Needs a platform uptake

File: src/System Application/App/Performance Profiler/src/PerfProfilerSchedulesList.Page.al [hunk 1]
```diff
@@ -26,8 +26,50 @@ page 1933 "Perf. Profiler Schedules List"
...
+            group("Profiling Status")
             {
+                Caption = 'Profiling Status';
+                AboutTitle = 'Profiling Status';
+                AboutText = 'See if profiling is enabled for the current user session.';
+
+                field("Profiling Enabled"; IsProfilingEnabled)
+                {
+                    Caption = 'Profiling Enabled';
+                    ToolTip = 'Shows if profiling is enabled for the current user session.';
```
Comment (NKarolak) [inline src/System Application/App/Performance Profiler/src/PerfProfilerSchedulesList.Page.al]: Tooltips should begin with "Specifies".
https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/user-assistance#guidelines-for-tooltip-text
(check all other new tooltips as well)

File: src/System Application/App/Performance Profiler/src/PerfProfilerSchedulesList.Page.al [hunk 2]
```diff
@@ -26,8 +26,50 @@ page 1933 "Perf. Profiler Schedules List"
...
+                    trigger OnDrillDown()
+                    var
+                        PerformanceProfileScheduler: Record "Performance Profile Scheduler";
+                        ScheduleCardPage: Page "Perf. Profiler Schedule Card";
+                    begin
+                        if (IsNullGuid(ActiveScheduleId)) then
+                            exit;
+
+                        if (not PerformanceProfileScheduler.Get(ActiveScheduleId)) then
+                            exit;
```
Comment (NKarolak) [inline src/System Application/App/Performance Profiler/src/PerfProfilerSchedulesList.Page.al]:
```suggestion
                        if IsNullGuid(ActiveScheduleId) then
                            exit;

                        if not PerformanceProfileScheduler.Get(ActiveScheduleId) then
                            exit;
```

**Review by NKarolak — CHANGES_REQUESTED** [review]
Nit: I prefer to have all properties alphabetically sorted. That way, it's easier to compare what is set and what is not between other fields or objects.

File: src/System Application/App/Performance Profiler/src/PerfProfilerSchedulesList.Page.al (hunk 2 above)
Comment (BazookaMusic) [reply src/System Application/App/Performance Profiler/src/PerfProfilerSchedulesList.Page.al]: Done

File: src/System Application/App/Performance Profiler/src/PerfProfilerSchedulesList.Page.al (hunk 1 above)
Comment (BazookaMusic) [reply src/System Application/App/Performance Profiler/src/PerfProfilerSchedulesList.Page.al]: Fixed in all the tooltips I could find

**Comment by BazookaMusic** [comment]
> Nit: I prefer to have all properties alphabetically sorted. That way, it's easier to compare what is set and what is not between other fields or objects.

I see the utility of this but I'd rather sort the properties I'd like to compare rather than spend all the time to sort properties manually for all fields/objects etc.

**Review by NKarolak — APPROVED** [review]

**Review by onbuyuka — APPROVED** [review]

**Review by onbuyuka — APPROVED** [review]

**Review by StefanSosic — APPROVED** [review]

**Pull request merged**